src/reducers/polling.js

```diff
@@ -45,8 +45,8 @@ const captchaRequired = (state) => (
 );
 
 const rateLimitExceeded = (state) => {
-  let newState = (state.get('interval') + 1000);
-  return state.set('interval', newState); // keep adding 1 second till we are not rate limited
+  const newState = (state.get('interval') + (3 * 1000));
```

Comment:
just realized my mistake..   newState wouldn't sound right.     how about  newInterval  or something that makes sense.